spec/Components/CioAutocomplete/CioAutocomplete.test.tsx

```diff
@@ -184,5 +184,45 @@ describe('CioAutocomplete Client-Side Rendering', () => {
         expect(suggestionCount).toBeInTheDocument();
       });
     });
+
+    it('Applies features sent in advancedParameters even if a custom UI variant is set that affects that feature', async () => {
```

Comment:
The test is failing 🤔 


<img width="1570" alt="Screenshot 2025-03-18 at 9 04 27 PM" src="https://github.com/user-attachments/assets/f2921585-1b29-4d86-875d-f228dda384ef" />